protocol/testutil/keeper/auth.go

```diff
@@ -41,6 +42,8 @@ func createAccountKeeper(
 		types.FeeCollectorName:            nil,
 		satypes.ModuleName:                nil,
 		perpetualstypes.InsuranceFundName: nil,
+		stakingtypes.BondedPoolName:       {types.Burner, types.Staking},
```

Comment:
For my understanding - what breaks if this is not added?

Reply:
[This](https://github.com/cosmos/cosmos-sdk/blob/main/x/staking/keeper/keeper.go#L129-L134) fails when instantiating new staking keeper(accountkeeper is passed to staking keeper)